cluster/distvalidator.go

```diff
@@ -75,3 +80,90 @@ func (v DistValidator) PublicKey() (*bls_sig.PublicKey, error) {
 func (v DistValidator) PublicShare(peerIdx int) (*bls_sig.PublicKey, error) {
 	return tblsconv.KeyFromBytes(v.PubShares[peerIdx])
 }
+
+// distValidatorJSONv1x1 is the json formatter of DistValidator for versions v1.0.0 and v1.1.0.
+type distValidatorJSONv1x1 struct {
+	PubKey              string   `json:"distributed_public_key"`
+	PubShares           [][]byte `json:"public_shares,omitempty"`
+	FeeRecipientAddress string   `json:"fee_recipient_address,omitempty"`
+}
+
+// distValidatorJSONv1x2 is the json formatter of DistValidator for versions v1.2.0 and later.
+type distValidatorJSONv1x2 struct {
+	PubKey              ethHex   `json:"distributed_public_key"`
+	PubShares           []base58 `json:"public_shares,omitempty"`
```

Comment:
why are we changing format of PubShares?
Since PubShares are internal public keys of individual co-validator we won't be using it anyways anywhere. Seems like a premature change.

we can keep it the same

Reply:
we are upgrading all base64 types to either base58 or hex. It is in the ticket.